packages/datadog_flutter_plugin/README.md

```diff
@@ -8,10 +8,10 @@ Datadog RUM SDK versions >= 1.4 support monitoring for Flutter 3.0+.
 ## Current Datadog SDK Versions
 
 [//]: # (SDK Table)
-
+        
```

Comment:
should it be just a new line?

Reply:
Yes probably. I'll leave it for now but I'll double check the releaser script.